### ISSUE TITLE
Retina assets.

### ISSUE THREAD

**Issue by jordwalke** [macoscope/CodePilot#53]
It seems that CodePilot doesn't render correctly on retina screens. The logo and others icons are blurry. Is this a known issue?

**Comment by justMaku** [comment]
Good catch! Will see if we still have the original files for those assets and if we do, we will try to make them retina compatible.
